CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.15.3 (2021-03-23)
+
+- Updated the change detection for `skyThemeIf` to use lifecycle hook. [#215](https://github.com/blackbaud/skyux-theme/pull/215)
```

Comment:
```suggestion
- Updated the change detection for `skyThemeIf` so that it uses a lifecycle hook. [#215](https://github.com/blackbaud/skyux-theme/pull/215)
```